### ISSUE TITLE
where is the FAM

### ISSUE THREAD

**Issue by lvzhengyi0204** [donnyyou/torchcv#138]

**Comment by lvzhengyi0204** [comment]
I want to know where is the Flow Alignment Module（FAM）. I
can not find it anywhere

**Comment by zimenglan-sysu-512** [comment]
+1, i want to see how FAM is implemented and how to do trt?

**Comment by zimenglan-sysu-512** [comment]
btw, @lxtGH   where to find the supplementary file?

**Comment by facetohard** [comment]
I think FAM is easy to implement.
`high_level_f = conv1_1(high_level) low_level_f = conv1_1(low_level)
 cat_f = torch.cat([high_level_f, low_level_f], dim=1)  flow = CBR(cat_f)`
And you can use warp ops refer to [flownet2-pytorch](https://github.com/NVIDIA/flownet2-pytorch/tree/master/networks/resample2d_package)

**Comment by lxtGH** [comment]
Sorry for late reply. Before the article is officially published, we do not consider releasing the code.

**Comment by facetohard** [comment]
@lxtGH Did I implement correctly?

**Comment by lxtGH** [comment]
Hello Everyone ! Here we upload the Resnet-18-based-SFNet. https://github.com/donnyyou/torchcv/blob/master/model/seg/nets/sfnet.py
